src/bundle/Resources/views/themes/admin/trash/list.html.twig

```diff
@@ -19,7 +20,54 @@
         <div class="px-0 ez-content-container">
             <section class="container mt-5">
                 {% include '@ezdesign/ui/page_title.html.twig' with { title: 'trash.headline'|trans|desc('Trash'), icon_name: 'trash' } %}
+                
+                {{ form_start(form_search, { attr: { class: 'ez-trash-search-form' } }) }}
+                    <div class="ez-trash-search-form__inputs-group px-4">
+                        {% set creator = form_search.vars.data ? form_search.vars.data.creator : '' %}
+                        
+                        {{ form_widget(form_search.content_type, {'attr': {'class': 'ez-trash-search-form__item'}}) }}
+                        {{ form_widget(form_search.section, {'attr': {'class': 'ez-trash-search-form__item'}}) }}
+                        {{ form_widget(form_search.trashed, {'attr': {'class': 'ez-trash-search-form__item'}}) }}
+
+                        <div class="ez-trash-search-form__item ez-trash-search-form__item--creator">
+                            {% set creator = form_search.vars.data ? form_search.vars.data.creator : '' %}
+                            <div class="ez-trash-search-form__input-wrapper">
+                                <input 
+                                    type="text"
+                                    class="form-control ez-trash-search-form__input"
+                                    data-content-type-identifiers="{{ user_content_type_identifier|join(',') }}"
+                                    value="{{ creator is not empty ? ez_content_name(creator) }}"
+                                    placeholder="{{ 'search.creator_input.placeholder'|trans|desc('Type creator\'s name') }}"
+                                    {{ creator is not empty ? 'disabled'  }}
+                                >
+                                <svg class="ez-icon ez-icon--dark ez-icon--medium ez-icon--reset">
```

Comment:
This should be inside a button and event listener should be attached to the button.